packages/screens/RWA/components/RWAScreenContainer/Header.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { useState } from "react";
+import { Pressable, View, ViewStyle } from "react-native";
+import { useSelector } from "react-redux";
+
+import contactsSVG from "../../../../../assets/icons/contacts.svg";
+import hamburgerCrossSVG from "../../../../../assets/icons/hamburger-button-cross.svg";
+import hamburgerSVG from "../../../../../assets/icons/hamburger-button.svg";
+import { BrandText } from "../../../../components/BrandText";
+import { SVG } from "../../../../components/SVG";
+import { SecondaryButton } from "../../../../components/buttons/SecondaryButton";
+import { ConnectWalletModal } from "../../../../components/connectWallet/ConnectWalletModal";
+import { BackButton } from "../../../../components/navigation/components/BackButton";
+import { SpacerRow } from "../../../../components/spacer";
+import { useRWASideBar } from "../../../../context/SidebarProvider";
+import useSelectedWallet from "../../../../hooks/useSelectedWallet";
+import { useTheme } from "../../../../hooks/useTheme";
+import { selectIsLightTheme } from "../../../../store/slices/settings";
+import { headerHeight, layout } from "../../../../utils/style/layout";
+
+export type HeaderProps = {
+  headerTitle: string;
+  onBackPress?: () => void;
+};
+
+export const HeaderMobile: React.FC<HeaderProps> = ({
+  headerTitle,
+  onBackPress,
+}) => {
+  const theme = useTheme();
+  const [isConnectWalletVisible, setIsConnectWalletVisible] = useState(false);
+  const selectedWallet = useSelectedWallet();
+  const { toggleSidebar, isSidebarExpanded } = useRWASideBar();
+
+  const toggleConnectWallet = () =>
+    setIsConnectWalletVisible(!isConnectWalletVisible);
+
+  return (
+    <View
+      style={{
+        ...HeaderContainerCStyle,
+        backgroundColor: theme.headerBackgroundColor,
+        borderBottomColor: theme.borderColor,
+      }}
+    >
+      <View style={HeaderRowCStyle}>
+        <View style={{ flex: 1, flexDirection: "row", alignItems: "center" }}>
+          {onBackPress && <BackButton onPress={onBackPress} />}
+          <BrandText
+            numberOfLines={1}
+            style={{ letterSpacing: -1, maxWidth: "75%", fontSize: 20 }}
```

Comment:
Why `maxWidth: "75%"` ? :o
```suggestion
            style={{ letterSpacing: -1, fontSize: 20 }}
```